src/core/ComponentCell/index.stories.tsx

```diff
@@ -0,0 +1,114 @@
+import { FormControlLabel } from "@mui/material";
+import { Args, Story } from "@storybook/react";
+import * as React from "react";
+import {
+  PrimaryText,
+  SecondaryText,
+  StyledTableData,
+} from "../BasicCell/style";
+import Checkbox from "../Checkbox";
+import InputToggle from "../InputToggle";
+import Tag from "../Tag";
+import ComponentCellRaw from "./index";
+
+const ComponentCell = (props: Args): JSX.Element => {
+  return (
+    <table>
+      <tbody>
+        <tr
```

Comment:
After merging the Row component, we need to update these &lt;tr&gt; to SDS &lt;Row&gt;.